client/src/templates/Challenges/classic/mobile-layout.tsx

```diff
@@ -73,6 +73,9 @@ class MobileLayout extends Component<MobileLayoutProps, MobileLayoutState> {
     // Unlike the desktop layout the mobile version does not have an ActionRow,
     // but still needs a way to switch between the different tabs.
 
+    const showToolPanel =
+      !usesMultifileEditor || (usesMultifileEditor && !hasEditableBoundaries);
```

Comment:
This could be simplified to check if `!hasEditableBoundaries`

Reply:
Thanks for the tip. Basically, I don't need `showToolPanel` any more so I've removed this completely.